pkg/analyser/analysis.go

```diff
@@ -189,3 +191,17 @@ func (a *Analysis) Summary() Summary {
 func (a *Analysis) Alerts() alerter.Alerts {
 	return a.alerts
 }
+
+func (a *Analysis) SortResources() {
+	a.differences = SortDifferences(a.differences)
+}
+
+func SortDifferences(diffs []Difference) []Difference {
+	sort.SliceStable(diffs, func(i, j int) bool {
```

Comment:
This does not sort the changed inside difference does it ?

Reply:
To make the output full predictable we should sort on `Change.Type` then `Change.Path` ? Or maybe only on `Change.Path` ?

Reply:
we could only sort on change.path but the type sort was added for readability at some point I think